frontend/packages/dev-console/integration-tests/testData/yamls/BuildConfig/buildconfig-with-strategy-source-source-binary.yaml

```diff
@@ -0,0 +1,106 @@
+kind: Build
+apiVersion: build.openshift.io/v1
+metadata:
+  annotations:
+    openshift.io/build-config.name: fruits-app-1
+    openshift.io/build.number: '1'
+    openshift.io/build.pod-name: fruits-app-1-1-build
+  resourceVersion: '338752'
+  name: fruits-app-1-1
+  uid: cd73cce1-75b4-484e-9b35-f6c9dc3ed8b6
+  creationTimestamp: '2021-05-04T07:45:14Z'
+  generation: 2
+  namespace: aut-form-edit-build-config
+  ownerReferences:
+    - apiVersion: build.openshift.io/v1
+      kind: BuildConfig
+      name: fruits-app-1
+      uid: 44291715-6933-4fac-82f1-481ea0fe7f29
```

Comment:
Drop this lines, they will be generated by kubernetes

```suggestion
```

Reply:
When I tried YAML on cluster it threw an error saying this uid is reuired so I've not removed this one